src/main/java/se/sundsvall/supportmanagement/integration/db/model/AttachmentEntity.java

```diff
@@ -29,7 +29,9 @@
 @Entity
 @Table(name = "attachment",
 	indexes = {
-		@Index(name = "idx_attachment_file_name", columnList = "file_name")
+		@Index(name = "idx_attachment_file_name", columnList = "file_name"),
+		@Index(name = "idx_attachment_municipality_id", columnList = "municipality_id"),
+		@Index(name = "idx_attachment_namespace", columnList = "namespace")
 	},
```

Comment:
Pretty sure none of these are being used.

Reply:
src/main/java/se/sundsvall/supportmanagement/integration/db/AttachmentRepository.java